app/src/main/java/com/firebase/ecommerce/feature_home/domain/HomeData.kt

```diff
@@ -7,9 +7,9 @@ import kotlinx.android.parcel.Parcelize
 data class HomeData(
     val email:String?=null,
     val password:String?=null,
-    val mobileNumber:String?=null,
+    var mobileNumber:String?=null,
     val gender:String?=null,
-    val userName:String?=null,
+    var userName:String?=null,
     val image:String="https://firebasestorage.googleapis.com/v0/b/ecommerce-b13bb.appspot.com/o/Profile%2Fprofile_image.jpg?alt=media&token=0b671803-1084-4bdc-8c36-206ecda5cab5"
```

Comment:
@Ankita123-oss Please provide default images with default values only, not with firebaseStorage default values